docs/source/quickstart/local-subgraphs.mdx

```diff
@@ -27,6 +27,10 @@ Import the `gql` tag into the file where you create your schema (`index.ts` for
 import gql from 'graphql-tag';
 ```
 
+```js title="index.js"
+import gql from 'graphql-tag';
```

Comment:
Do we need to include this because the default JS block ends up blank? Just curious!

Reply:
Yes it does! I meant to ask DXE about this but figured I'd include the JS blocks in the meantime.

Reply:
There's a way to do this which is actually documented... you just have to know that the compilation is implemented by @trevorblades 's package `remark-typescript` and read its [README](https://github.com/trevorblades/remark-typescript). (Maybe this should go in the `docs` README too.) Add `// preserve-line` to the import that you don't want to lose.

Reply:
Oh, that's so cool! Thank you for the pro-tip!